### PR TITLE
gui: Package attribution and copyright bumps (fixes #3861)

### LINKED ISSUE #3861: Update the list of packages and their copyrights in the about modal
It's pretty out of date.

### PR DESCRIPTION
**Pull request by ProactiveServices**

### CODE REVIEW

File: gui/default/syncthing/core/aboutModalView.html [hunk 1]
```diff
@@ -20,15 +20,26 @@ <h1 class="text-center">
     <p translate>Syncthing includes the following software or portions thereof:</p>
     <ul class="list-unstyled two-columns">
       <li><a href="https://golang.org">The Go Programming Language</a>, Copyright &copy; 2012 The Go Authors.</li>
+      <li><a href="https://github.com/sasha-s/go-deadlock">sasha-s/go-deadlock</a>, Copyright &copy; 2016 sasha-s</li>
+      <li><a href="https://github.com/rcrowley/go-metrics">rcrowley/go-metrics</a>, Copyright &copy; 2012 Richard Crowley.</li>
+      <li><a href="https://github.com/oschwald/geoip2-golang">oschwald/geoip2-golang</a>, <a href="https://github.com/oschwald/maxminddb-golang">oschwald/maxminddb-golang</a>, Copyright &copy; 2015 <a href="mailto:oschwald@gmail.com">Gregory J. Oschwald</a>.</li>
```
Comment (calmh) [inline gui/default/syncthing/core/aboutModalView.html]: Relay pool server only

File: gui/default/syncthing/core/aboutModalView.html [hunk 2]
```diff
@@ -20,15 +20,26 @@ <h1 class="text-center">
...
+      <li><a href="https://github.com/onsi/ginkgo/">onsi/ginkgo/</a>, <a href="https://github.com/onsi/gomega/">onsi/gomega/</a> Copyright &copy; 2013-2014 Onsi Fakhouri.</li>
```
Comment (calmh) [inline gui/default/syncthing/core/aboutModalView.html]: Tests for some other package only

File: gui/default/syncthing/core/aboutModalView.html [hunk 3]
```diff
@@ -20,15 +20,26 @@ <h1 class="text-center">
...
+      <li><a href="https://github.com/minio/sha256-simd">minio/sha256-simd</a>, Copyright &copy; 2016 Minio, Inc.</li>
+      <li><a href="https://github.com/lib/pq">lib/pq</a>, Copyright &copy; 2011-2013 &apos;pq&apos; Contributors Portions Copyright &copy; 2011 Blake Mizerany.</li>
```
Comment (calmh) [inline gui/default/syncthing/core/aboutModalView.html]: Relay pool server only

File: gui/default/syncthing/core/aboutModalView.html [hunk 4]
```diff
@@ -20,15 +20,26 @@ <h1 class="text-center">
...
+      <li><a href="https://github.com/jackpal/gateway">jackpal/gateway</a>, Copyright &copy; 2010 Jack Palevich.</li>
+      <li><a href="https://github.com/gogo/protobuf">gogo/protobuf</a>, Copyright &copy; 2013 The GoGo Authors.</li>
+      <li><a href="https://github.com/gobwas/glob">gobwas/glob</a>, Copyright &copy; 2016 Sergey Kamardin.</li>
+      <li><a href="https://github.com/cznic/lldb">cznic/lldb</a>, Copyright &copy; 2014 The lldb Authors.</li>
```
Comment (calmh) [inline gui/default/syncthing/core/aboutModalView.html]: Discovery server only

**Comment by ProactiveServices** [comment]
How about a "-version" command line argument which would print version number and 3rd-party attributions? I think I could stretch my programming expertise that far ;-)

**Comment by calmh** [comment]
I think typically it's only included in READMEs in the package, which we don't really do for those other commands but could. A `-version` or `-copyright` flag would be fine as well although I can't find another CLI program right now that includes that output so I think it may be overkill.

File: cmd/strelaypoolsrv/README.md [hunk 5]
```diff
@@ -13,3 +13,9 @@ If you still want to run it, you can run `go get github.com/syncthing/relaypools
 from the build server.
 
 See `relaypoolsrv -help` for configuration options.
+
+##### Third-party attributions
+
+[oschwald/geoip2-golang](https://github.com/oschwald/geoip2-golang), [oschwald/maxminddb-golang](https://github.com/oschwald/maxminddb-golang), Copyright (C) 2015 [Gregory J. Oschwald](mailto:oschwald@gmail.com).
+
+[lib/pq](https://github.com/lib/pq)</a>, Copyright (C) 2011-2013 'pq' Contributors Portions Copyright (C) 2011 Blake Mizerany.
```
Comment (AudriusButkevicius) [inline cmd/strelaypoolsrv/README.md]: I don't think this is used here. It's probably in discovery server. Also zinc(or something like that)/ql is used for discovery too. Which is in memory query language.

Comment (calmh) [inline cmd/strelaypoolsrv/README.md]: Indeed, I was wrong about this before:

```
jb@unu:~/s/g/s/syncthing $ git grep cznic/ql | grep cmd
cmd/stdiscosrv/ql.go:	"github.com/cznic/ql"
jb@unu:~/s/g/s/syncthing $ git grep lib/pq | grep cmd
cmd/stdiscosrv/psql.go:	_ "github.com/lib/pq"
jb@unu:~/s/g/s/syncthing $ 
```

**Comment by calmh** [comment]
@st-review merge

**Comment by st-review** [comment]
:ok_hand: Merged as c953cdc375fe039d9156f862a448986c9db1d141. Thanks, @ProactiveServices!

**Pull request closed without merging**